### ISSUE TITLE
fmtlib places files outside of `.data` when not linking against Python

### ISSUE THREAD

**Issue by JCGoran** [BlueBrain/nmodl#1153]
When using `-DLINK_AGAINST_PYTHON=FALSE` (for instance, for building wheels), Eigen and related libs are placed in `nmodl/.data`, but fmtlib doesn't respect this and stubbornly puts items in other places. Below is an excerpt from the `install_manifest.txt` file (tested with latest master):

```plaintext
/lib/libfmt.a
/include/fmt/args.h
/include/fmt/chrono.h
/include/fmt/color.h
/include/fmt/compile.h
/include/fmt/core.h
/include/fmt/format.h
/include/fmt/format-inl.h
/include/fmt/os.h
/include/fmt/ostream.h
/include/fmt/printf.h
/include/fmt/ranges.h
/include/fmt/std.h
/include/fmt/xchar.h
/lib/cmake/fmt/fmt-config.cmake
/lib/cmake/fmt/fmt-config-version.cmake
/lib/cmake/fmt/fmt-targets.cmake
/lib/cmake/fmt/fmt-targets-release.cmake
/lib/pkgconfig/fmt.pc
/nmodl/.data/lib/libpywrapper.dylib
/nmodl/_nmodl.cpython-311-darwin.so
/nmodl/.data/include/newton/newton.hpp
/nmodl/.data/include/Eigen/CholmodSupport
/nmodl/.data/include/Eigen/StdVector
/nmodl/.data/include/Eigen/StdDeque
... other files in /nmodl/.data/
```

I don't know if this is intended or not, but it's somewhat annoying since the wheel built with `scikit-build-core` (see #1147) has `lib` (or `lib64` on Linux) and `include` directories along with the `nmodl` one.

**Comment by 1uc** [comment]
We don't install `fmt` anymore.

**Comment by JCGoran** [comment]
For future readers, this was fixed by #1193
